Common/include/CConfig.hpp

```diff
@@ -9617,10 +9620,73 @@ class CConfig {
    */
   unsigned short GetKind_Grad_Linear_Solver_Prec(void) const { return Kind_Grad_Linear_Solver_Prec; }
 
-    /*!
+  /*!
    * \brief Get max number of iterations of the for the gradient smoothing.
    * \return Max number of iterations of the linear solver for the gradient smoothing.
    */
   unsigned long GetGrad_Linear_Solver_Iter(void) const { return Grad_Linear_Solver_Iter; }
 
+  /*!
+   * \brief Get parsed SST option data structure.
+   * \return SST option data structure.
+   */
+  SST_ParsedOptions GetSSTParsedOptions(void) const {return sstParsedOptions; }
+
+  /*!
+   * \brief function to parse SST options.
+   * \param[in] SST_Options - Selected SST option from config.
+   * \param[in] nSST_Options - Number of options selected.
+   */
+  SST_ParsedOptions ParseSSTOptions(const SST_OPTIONS* SST_Options, unsigned short nSST_Options){
```

Comment:
You also need to pass either the rank or a "is_master" boolean, otherwise the messages will be printed too many times.

Reply:
I've added the rank==MASTER_NODE. This routine is still called 3 times, so we see the messages 3 times (because setpostprocessing is called 3 times?)

Reply:
Probably.
Ok, but move this function back to option_structure please...
And remove the files that were committed by mistake, it makes this difficult to navigate.